pymc_marketing/clv/models/gamma_gamma.py

```diff
@@ -412,89 +404,3 @@ def build_model(self):
             pm.Gamma(
                 "spend", p, nu[self.data["customer_id"]], observed=z, dims=("obs",)
             )
-
-    def _summarize_mean_data(self, customer_id, individual_transaction_value):
```

Comment:
Just double-checking :D : Why are you getting rid of all these methods?

Reply:
`_summarize_mean_data` is basically doing the same thing as `clv.utils.rfm_summary`, and also reassigns customer IDs which doesn't sit well with me. This was a workaround for the fact `GammaGammaModelIndividual` is trained on individual transaction values, yet the predictive methods require frequency/recency summarization.

I didn't create this model originally, but it seems to have been improvised. It may be possible to to take more of a simulation approach like in `ShiftedBetaGeoModelIndividual`, so I'll get an issue created.